src/main/resources/com/xebialabs/xlrelease/ci/Credential/config.jelly

```diff
@@ -10,6 +10,11 @@
     <f:entry title="${%Password}" field="password">
         <f:password/>
     </f:entry>
+    <f:optionalBlock inline="true" field="useGlobalCredential" title="Use stored credentials" checked="${instance.showGolbalCredentials()}">
+        <f:entry title="${%Credentials}" field="credentialsId">
+            <c:select/>
+        </f:entry>
+    </f:optionalBlock>
```

Comment:
question: When selecting this checkbox should the previous fields (name, username and password) mark as not editable?